muon-app/src/main/java/muon/app/ui/components/session/files/view/FolderView.java

```diff
@@ -145,8 +195,22 @@ public int compare(Object o1, Object o2) {
 		sorter.setComparator(5, new Comparator<>() {
 			@Override
 			public int compare(Object o1, Object o2) {
-				String s1 = (String) ((FileInfo) o1).getPermissionString().toString();
-				String s2 = (String) ((FileInfo) o2).getPermissionString().toString();

```

Comment:
`getPermissionString()` methoud should be replaced with `.getUser()` which I done in the PR